### DISCUSSION TITLE
I didn't get my xch

### DISCUSSION THREAD

**Discussion by mindas23** [Chia-Network/chia-blockchain#14318] [Support]
I transferred my existing xch 10,139 from the hobby to your wallet and entered the address found in the wallet Receive Address xch1awqx7f5x9feph4h0lqsdjy77e3aywk3pec7qsjugq3zt5q2s698smlyp3h but I did not receive my token. TxID 211d94a9f86e0d667cb0e7a66a2ef5338310a5e84290bc142bd5512210d328e5

can you help me
mind23 3507005215

**Comment by mindas23** [comment]
mindas23 3507005215

**Comment by Bodman1** [comment]
this transaction was made on January 18, 2023, 7:22 am, at 33 chia, that evening I also transferred 27.9482 chia kucoin from the exchange to my wallet, in the support of the exchange they wrote to me that the coins were sent to the wallet, but on the wallet I didn't see them. transaction data

The withdrawal you made on 2023-01-19 03:04:26 (UTC+08:00) was successful.
Token: | XCH
Amount: | 27.9482
Address: | xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72

**Comment by mindas23** [comment]
something is wrong here

I asked for your help as I have not received xch
but what you sent is not my request

I don't believe at all that you can do anything more and I didn't write


2023-01-22, sk, 20:18 Bogdan ***@***.***> rašė:

**Reply by Bodman1** [reply]
> something is wrong here I asked for your help as I have not received xch but what you sent is not my request I don't believe at all that you can do anything more and I didn't write 2023-01-22, sk, 20:18 Bogdan ***@***.***> rašė:
> […](#)

sorry, I have the same problem as you, I am also looking for ways to solve it, if I find a way to solve it, I will write to you

**Comment by mindas23** [comment]
Vsio caraso Bogdan

Mnie niekto neatvetil ne po poctie ne po forumu niekak nemagu poniat kak
normalna padkliucyt etat chia kasyliok mnie vsiegda pyset sto
oflain probaval iskat yuotube no tam vsie staroje video jiesli sto uznaju
toze napysu.
Izvini za bukvy nieumieju pysat po ruski.


2023-01-24, an, 20:48 Bogdan ***@***.***> rašė:

**Comment by mindas23** [comment]
Hello, I found a solution, my chia coins appeared as I downloaded a
mobile app, if you are interested, write


2023-01-24, an, 22:45 Mindaugas Matulevičius ***@***.***>
rašė:
